src/com/hms_networks/americas/sc/fileutils/FileAccessManager.java

```diff
@@ -25,31 +26,41 @@
    * @throws IOException if unable to access or read file
    */
   public static String readFileToString(String filename) throws IOException {
-    // Create input stream and buffered reader
-    InputStream inputStream = new FileInputStream(filename);
-    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+    // If file does not exist, return null
+    File file = new File(filename);
+    String returnVal;
```

Comment:
if you set returnVal to null here you dont need the else

Reply:
changed